packages/material-ui/src/DialogContentText/DialogContentText.spec.tsx

```diff
@@ -16,13 +16,9 @@ const DialogContentTextTest = () => {
       <DialogContentText align="inherit" color="textPrimary" />
       <DialogContentText align="inherit" color="textSecondary" />
       <DialogContentText align="inherit" color="error" />
-      {/* @ts-expect-error */}
-      <DialogContentText display="incorrectValue" />
```

Comment:
This test needs to stay and a comment added that this is unintended. This you verify if IntelliSense is at least still working?

Reply:
Intellisense works as expected, with all values appropriate for the prop. I will leave it with a comment then

Reply:
Actually, something is off, let me double check the typings again.

Reply:
Nope, it doesn't work at all. I will try to use the typings we have on the `sx` prop..

Reply:
If it was working in JSDOM it would be even better. I was thinking of something similar to https://github.com/styled-components/jest-styled-components.

Reply:
I've improved the typings with https://github.com/mui-org/material-ui/pull/24496/commits/085eb98611a5b51d33c18fa0d754cff4e02d38ff

Intellisense is working as expected:

![image](https://user-images.githubusercontent.com/4512430/105161247-732f8d80-5b11-11eb-9e83-37f93275b10f.png)